people/peter.json

```diff
@@ -0,0 +1,11 @@
+{
+  "key": "peter",
+  "name": "Peter Nutter",
+  "nick": "Peter",
+  "email": "peter.nutter@mild.blue",
+  "title_cs": "JUNIOR DEVELOPER",
+  "title_en": "JUNIOR DEVELOPER",
+  "short_desc_en": "Peter studies Mathematical Informatics at the Faculty of Nuclear Sciences and Physical Engineering at CTU. At Mild Blue, he works as a junior developer.",
+  "short_desc_de": "Peter studiert Mathematische Informatik an der Fakultät für Kernforschung und Physikalisches Engineering an der Tschechischen Technischen Universität. Bei Mild Blue arbeitet er als Junior Developer.",
+  "short_desc_cs": "Peter studuje Matematickou informatiku na Fakultě jaderné a fyzikálně inženýrské na ČVUT. V Mild Blue pracuje na pozici junior developer"

```

Comment:
tak kdo to okopiroval od koho? Kazdopadne to same co u Mateje, zkus to trochu rozvest